GetIntoTeachingApi/Models/TeacherTrainingAdviserSignUp.cs

```diff
@@ -25,17 +27,26 @@ public class TeacherTrainingAdviserSignUp
         public int? PlanningToRetakeGcseMathsAndEnglishId { get; set; }
         public int? PlanningToRetakeGcseScienceId { get; set; }
 
+        [SensitiveData]
         public string Email { get; set; }
+        [SensitiveData]
         public string FirstName { get; set; }
+        [SensitiveData]
         public string LastName { get; set; }
         [SwaggerSchema(Format = "date")]
+        [SensitiveData]
         public DateTime? DateOfBirth { get; set; }
+        [SensitiveData]
         public string TeacherId { get; set; }
         public string DegreeSubject { get; set; }
+        [SensitiveData]
         public string Telephone { get; set; }
+        [SensitiveData]
         public string AddressLine1 { get; set; }
+        [SensitiveData]
         public string AddressLine2 { get; set; }
         public string AddressCity { get; set; }

```

Comment:
Any reason not to include AddressCity as well?

Reply:
I figured that `AddressCity` isn't personally identifiable (arguably `AddressPostcode` as well but I think some postcode regions can be very small) and it may help with debugging if left in -- happy to take it out if you disagree, but I'll do that in a follow up PR 👍